up_client_socket/cpp/tst.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+from threading import Thread
+# from uprotocol.proto.uattributes_pb2 import UMessageType, UPriority
+# from uprotocol.proto.ustatus_pb2 import UCode
+from dispatcher.dispatcher import Dispatcher
+
```

Comment:
The linter wants these lines changed:
```suggestion
from dispatcher.dispatcher import Dispatcher
from threading import Thread

```